hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -480,6 +481,8 @@ private void complete(Object value) {
     private void handleRetry(Object cause) {
         context.retryCount.inc();
 
+        context.logger.info("Retrying invocation: " + toString() + ", Reason: " + cause);
```

Comment:
1: Do we want to do this on info? Could generate a lot of noise.
2: I don't see a logging guard (could be that it is there; but this overview isn't showing it).